### PR TITLE
Fix storybook border style

### PR DESCRIPTION
**Pull request by oorestisime**
#### What does this PR do?
Fixes storybook 
#### What testing has been done on this PR?
storybook
#### How should this be manually tested?
storybook
#### What are the relevant issues?
#2959
#### Do the grommet docs need to be updated?
Docs are ok, grommet-site has an issue
#### Should this PR be mentioned in the release notes?
no
#### Is this change backwards compatible or is it a breaking change?
yes

### CODE REVIEW

File: src/js/components/Box/box.stories.js [hunk 1]
```diff
@@ -130,7 +130,11 @@ const BorderBox = () => (
           'inset',
           'outset',
         ].map(type => (
-          <Box key={type} pad="small" border={{ type, size: 'medium' }}>
+          <Box
+            key={type}
+            pad="small"
+            border={{ type, size: 'medium', style: type }}
```
Comment (ericsoderberghp) [inline src/js/components/Box/box.stories.js]: Two questions:

I don't see either `type` or `style` in the the documentation for Box https://v2.grommet.io/box#border, should we have it there?

Why do we need both `type` and `style` keys?

Comment (oorestisime) [reply src/js/components/Box/box.stories.js]: oups the type is just leftover i forgot to remove. on the other hand i don't know why the border style is not appearing on the grommet-site. it is present on the doc.js file.

Comment (ShimiSun) [reply src/js/components/Box/box.stories.js]: https://github.com/grommet/grommet-site/pull/101
updating the grommet site.

Comment (ShimiSun) [reply src/js/components/Box/box.stories.js]: but I kind of like this PR better for updating grommet-site https://github.com/grommet/grommet-site/pull/100/files

Comment (oorestisime) [reply src/js/components/Box/box.stories.js]: Removed the extra type prop

**Pull request merged**